Makefile

```
@@ -13,6 +13,11 @@ ifndef UNAME_M
 UNAME_M := $(shell uname -m)
 endif
 
+ARCH_LINUX := $(shell grep -e "Arch Linux" -e "ID_LIKE=arch" /etc/os-release)
```

I forgot to keep this:

```suggestion
ARCH_LINUX := $(shell grep -e "Arch Linux" -e "ID_LIKE=arch" /etc/os-release 2>/dev/null)
```